snapcraft/cli/lifecycle.py

```diff
@@ -273,20 +273,13 @@ def pack(directory, output, **kwargs):
 
 @lifecyclecli.command()
 @click.argument("parts", nargs=-1, metavar="<part>...", required=False)
-@click.option(
-    "--step",
```

Comment:
The ability to clean specific steps is SUPER helpful. For example, it's making debugging this PR hard :) .

Reply:
yes, read the bug description